client/post-editor/editor-action-bar/index.jsx

```diff
@@ -68,7 +68,13 @@ export default React.createClass( {
 		return (
 			<div className="editor-action-bar">
 				<div className="editor-action-bar__first-group">
-					{ multiUserSite && <EditorAuthor post={ this.props.post } isNew={ this.props.isNew } /> }
+					{ multiUserSite &&
+						<AsyncLoad
```

Comment:
Nice thing about this is that the code for `<EditorAuthor />` will now never be loaded for single user sites 😄